src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/AgoraStatus.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.agora.server.dataaccess
+
+import org.broadinstitute.dsde.agora.server.dataaccess.mongo.AgoraMongoClient._
+import org.broadinstitute.dsde.agora.server.dataaccess.permissions.AgoraEntityPermissionsClient._
+
+class AgoraStatus {
+  def status(): (Boolean, String) = {
+    val (up, message) = getMongoDBStatus
```

Comment:
the pattern for getMongoDBStatus is different than sqlDBStatus, the former does not throw and exception but the latter does.

I would change the signature to something like

checkMongoConnection(): Try[Unit]
checkSqlConnection(): Try[Unit]

I don't really like Unit but the return value on success does not really matter. When there is an exception you can extract the message from the failure. Then you can do something like

val mongo = checkMongoConnection()
val sql = checkSqlConnection()

val errors = Set(("mongo error", mongo), ("sql error", sql)).collect { case (label, Failure(t)) => s""""$label": "$t.message""""
if (errors.isEmpty) {
  """{"status": "up"}"""
} else {
  s"""{ "status": "down", ${errors.mkString(", ") }"""
}

That also centralizes all the json ick